File: models/ranking.go
package models

import (
	"time"
)

type UserRanking struct {
	UserID      string    `json:"user_id"`
	RankingID   string    `json:"ranking_id"`
	Name        string    `json:"name"`
	Description string    `json:"description"`
	Year        int       `json:"year"`
	Ranking     string    `json:"ranking"`
	GroupIDs    []string  `json:"group_ids"`
	CreatedAt   time.Time `json:"created_at"`
	UpdatedAt   time.Time `json:"created_at"`
}

File: EurovisionRanker.Api/EurovisionRanker.Api/Models/Ranking.cs
namespace EurovisionRanker.Api.Models
{
    public class Ranking
    {
        public Guid Id { get; set; }
        public Guid UserId { get; set; }
        public required string Name { get; set; }
        public string? Description { get; set; }
        public int Year { get; set; }

        // The raw URL-encoded string 
        public required string RankingString { get; set; }

        public DateTime UpdatedAt { get; set; }
    }
}
